lib/Menu.php

```diff
@@ -279,32 +250,33 @@ public function find_parent_item_in_menu( $menu_items, $parent_id ) {
 	 * @return array
 	 */
 	protected function order_children( $items ) {
-		$index = array();
-		$menu = array();
-		$wp_post_menu_item = null;
+		$items_by_id = [];
+		$menu        = [];
+
 		foreach ( $items as $item ) {
 			if ( isset($item->title) ) {
 				// Items from WordPress can come with a $title property which conflicts with methods
 				$item->__title = $item->title;
 				unset($item->title);
 			}
+
+			// Check it we're working with a post
 			if ( isset($item->ID) ) {
-				if ( is_object($item) && get_class($item) == 'WP_Post' ) {
-					$wp_post_menu_item = $item;
-					$item = new $this->PostClass($item);
-				}
-				$menu_item = $this->create_menu_item($item);
```

Comment:
I guess the `create_menu_item()` method can be removed, then?

Reply:
Removed!